pkg/virt-controller/watch/vmi/vmi.go

```diff
@@ -954,7 +954,7 @@ func (c *Controller) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod, da
 		// do not return; just log the error
 	}
 
-	err := c.backendStorage.CreateIfNeededAndUpdateVolumeStatus(vmi)
+	backendStoragePVCName, err := c.backendStorage.CreateIfNeededAndUpdateVolumeStatus(vmi)
```

Comment:
This is indeed where the PVC might get created. I don't quite understand your point here, but if we stop passing the PVC name around it might not be relevant anymore.